client/state/ui/test/selectors.js

```diff
@@ -241,4 +242,55 @@ describe( 'selectors', () => {
 			expect( siteSection ).to.be.true;
 		} );
 	} );
+
+	describe( '#isSectionIsomorphic()', () => {
+		test( 'should return false if there is no section currently selected', () => {
+			const selected = isSectionIsomorphic( {
+				ui: {
+					section: false,
+				},
+			} );
+
+			expect( selected ).to.be.false;
+		} );
+
+		test( 'should return true if current section is isomorphic', () => {
+			const section = {
+				enableLoggedOut: true,
+				group: 'sites',
+				isomorphic: true,
+				module: 'my-sites/themes',
+				name: 'themes',
+				paths: [ '/themes' ],
+			};
+
+			const selected = isSectionIsomorphic( {
+				ui: { section },
+			} );
+
+			expect( selected ).to.be.true;
+		} );
+	} );
```

Comment:
`yarn run test` here gives me an error: `    ReferenceError: isSectionIsomorphic is not defined`.  Maybe this was a reference copy/paste left in?

Reply:
Thanks for pointing this out, it should be something with a wrong merge conflict resolution. This code is not mine nor it exists on master, I removed it.
Not sure why didn't I try tests in this branch.. Now `yarn jest -c=test/client/jest.config.js client/state/ui/test/selectors.js` passes